### DISCUSSION TITLE
[Question] Authorized realtime connection canceled after 5m

### DISCUSSION THREAD

**Discussion by shynome** [pocketbase/pocketbase#1296] [Q&A] [answered]
Every realtime connection is canceled after 5m, I want to keep authorized realtime connection is to keep connected

https://github.com/pocketbase/pocketbase/blob/ca528cef032df7ffccefbaca9a3a65dc00ae7c62/apis/realtime.go#L101-L109

**Comment by ganigeorgiev** [comment, marked as answer]
Only **inactive/idle** realtime connections should be cancelled after 5 mins, aka. connections that didn't have any messages within that period.

This usually shouldn't be a problem because the SDKs will handle the reconnect for you seamlessly (_if the server is still online and responding_).

**Reply by shynome** [reply]
Thanks for your reply, I have implemented reconnection in [Go SDK](https://github.com/r--w/pocketbase/pull/19)
